frontend-instructor/src/main.jsx

```
@@ -1,48 +1,53 @@
       },
       {
-        path: "/add-courses",
-        element: <AddCourse />, 
+        path: '/add-courses',
+        element: <AddCourse />
       },
-    ],
-  },
+      {
+        path: '/analytics/:courseId',
+        element: <CourseAnalytics></CourseAnalytics>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)